Cargo.toml

```diff
@@ -12,7 +12,7 @@ crate-type = ["cdylib"]
 [dependencies]
 log = "0.4"
 itertools = "0.8"
-ironoxide = "0.10.1"
+ironoxide = { git = "https://github.com/IronCoreLabs/ironoxide.git", branch = "17-changes-for-scala"}
```

Comment:
don't forget to depend on a released version

Reply:
Done